### PR TITLE
Port to react

### PR DESCRIPTION
**Pull request by iefserge**
T = major

### CODE REVIEW

File: lib/text.js [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
+    if (React.isValidElement(el)) {
+      el = React.cloneElement(el, { key: index });
+    }
+
     return el;
   }
 
   if (item.type === 'linebreak') {
-    return <br/>;
+    return <br key={index} />;
```
Comment (ellell) [inline lib/text.js]: What's with the key?

**Comment by ellell** [comment]
@iefserge Looks great!! Just one minor question.

File: lib/text.js (hunk 1 above)
Comment (iefserge) [inline lib/text.js]: @ellell react requires all arrays of elements to have a unique key for each element

Comment (ellell) [inline lib/text.js]: ok, thanks 👍

**Pull request merged**